plugins/modules/ome_device_quick_deploy.py

```diff
@@ -519,6 +519,9 @@ def check_mode_validation(module, deploy_data):
         module.exit_json(msg=NO_CHANGES_FOUND, quick_deploy_settings=deploy_data)
     req_payload.update(resp_filter_data)
     req_payload.update(req_data_filter)
+    prefix_length = req_payload.get("PrefixLength")
```

Comment:
can we check if we can
initalize req_data["PrefixLength"] = "" in line 469
and add an and condition to existing if condition in line 470 if deploy_options.get("ipv6_prefix_length") !=0 to populate req_data["PrefixLength"] instead?